junit-tests/src/test/java/org/junit/gen5/engine/DummyTestEngineDescriptor.java

```diff
@@ -10,20 +10,21 @@
 
 package org.junit.gen5.engine;
 
-class DummyTestDescriptor extends AbstractTestDescriptor implements Leaf<DummyEngineExecutionContext> {
-
-	public static final String ENGINE_ID = "dummy";
-
+class DummyTestEngineDescriptor extends AbstractTestDescriptor implements Leaf<DummyEngineExecutionContext> {
```

Comment:
Based on the unique id of this dummy, it represents a test engine node. That is why I renamed it. I agree the new name does not reflect it either. `implements Leaf<bla>` was added after I changed the name, so the intent of this dummy might have changed. I can rename it back to what it was. Still I think we should talk about a naming convention for test doubles.